agent/php_user_instrument.c

```diff
@@ -67,7 +70,8 @@ int nr_zend_call_orig_execute_special(nruserfn_t* wraprec,
       wraprec->special_instrumentation(wraprec, segment,
                                        NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
     } else {
-      NR_PHP_PROCESS_GLOBALS(orig_execute)(NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
+      NR_PHP_PROCESS_GLOBALS(orig_execute)
+      (NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
```

Comment:
[This comment](https://github.com/newrelic/newrelic-php-agent/pull/541#discussion_r988323293) applies here too.